runtime/src/main/cpp/ObjCMMAPI.h

```diff
@@ -12,6 +12,22 @@
 
 extern "C" ALWAYS_INLINE void Kotlin_ObjCExport_releaseAssociatedObject(void* associatedObject);
 
+namespace konan {
```

Comment:
I think we'll need to specify how we use namespaces.

Reply:
Yes.